src/uct/ib/base/ib_device.c

```diff
@@ -629,39 +620,42 @@ int uct_ib_device_test_roce_gid_index(uct_ib_device_t *dev, uint8_t port_num,
     return 1;
 }
 
-static ucs_status_t uct_ib_device_set_roce_gid_index(uct_ib_device_t *dev,
-                                                     uint8_t port_num,
-                                                     uint8_t *gid_index)
+ucs_status_t uct_ib_device_set_roce_info(uct_ib_device_t *dev,
+                                         uint8_t port_num,
+                                         uint8_t *gid_index,
+                                         uct_ib_roce_version_desc_t *roce_ver)
 {
     static const uct_ib_roce_version_desc_t roce_prio[] = {
-        {2, 0, AF_INET},
-        {2, 0, AF_INET6},
-        {1, 0, AF_INET},
-        {1, 0, AF_INET6}
+        {UCT_IB_FLAG_ROCE_V2, AF_INET},
+        {UCT_IB_FLAG_ROCE_V2, AF_INET6},
+        {UCT_IB_FLAG_ROCE_V1, AF_INET},
```

Comment:
@yosefe ROCE_V1_5 isn't present in the priority list. Is this intentionally or need to add?

Reply:
it's intentional, no need to add